guides/common/modules/proc_generating-a-report-for-installable-and-applicable-errata.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Generating_a_Report_for_Installable_and_Applicable_Errata_{context}"]
+= Generating a Report for Installable and Applicable Errata
+
+Use the following procedure to view installable and applicable errata on managed hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Click *Generate* for the *Host {endash} Applicable Errata* template.
+. In the *Installability* drop down menu, select *Applicable* to show all applicable errata or *Installable* to limit the report exclusively to errata that are accessible in the Content View environments of your host that can be installed.
+. Optional: To schedule a report, click the icon to the right of the *Generate at* field and choose the date and time you want for the generated report.
+. Optional: To send a report to an e-mail address, select the *Send report via e-mail* checkbox, and in the *Deliver to e-mail addresses* field, enter the required e-mail address.
+. Click *Generate* to generate a CSV file that contains the report.
+After {Project} has created the report, your browser automatically downloads it.
++
+If you have selected the *Send report via e-mail* checkbox, the report is sent to your e-mail address.
```

Comment:
I would avoid using present perfect tense here and on line 13 if it makes sense to you.